cicd/azion-massive-redirect-ptbr.json

```diff
@@ -282,5 +282,833 @@
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/schedule-blocking/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/scheduled-blocking/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentation/products/guides/angular-boilerplate/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/angular-boilerplate/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/send-event/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/send-event-to-endpoint/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/comece-com-um-template/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/comecar-com-um-template/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/build/trabalhe-com-rules-engine/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/build/trabalhar-com-rules-engine/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/permissoes-mktp/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/permissoes-marketplace/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/limitpayload/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/limit-payload-size/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/load-balancer/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/build/edge-application/load-balancer/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/edge_application/main-settings/",
+        "moved": "https://www.azion.com/en/documentation/products/edge-application/build/main-settings/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/build/build-an-application/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/build/criar-uma-aplicacao/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/produtos/guias/secure/digital-certificates/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/secure/certificado-digital/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/docs/casos-de-uso/massive-redirect",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/configurar-massive-redirect/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/casos-de-uso/massive-redirect",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/configurar-massive-redirect/"
+    },
+    {
+        "from": "https://www.azion.com/pt-br/documentacao/casos-de-uso/massive-redirect/",
+        "moved": "https://www.azion.com/pt-br/documentacao/produtos/guias/configurar-massive-redirect/"
```

Comment:
ADD
https://www.azion.com/pt-br/documentacao/produtos/guias/how-to-redirecionamento-massivo/ > https://www.azion.com/pt-br/documentacao/produtos/guias/massive-redirect-integration/

Reply:
https://www.azion.com/pt-br/documentacao/produtos/guias/massive-redirect/ > https://www.azion.com/pt-br/documentacao/produtos/guias/massive-redirect-integration/

Reply:
Ok no redirects-pt